resources/strings/src/commonMain/kotlin/dev/sasikanth/rss/reader/resources/strings/DeTwineStrings.kt

```diff
@@ -0,0 +1,149 @@
+/*
+ * Copyright 2024 Sasikanth Miriyampalli
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package dev.sasikanth.rss.reader.resources.strings
+
+import cafe.adriel.lyricist.LyricistStrings
+
+@LyricistStrings(languageTag = Locales.DE, default = false)
+val DeTwineStrings =
+    TwineStrings(
+        appName = "Twine",
+        postSourceUnknown = "Unbekannt",
+        buttonAll = "Alle",
+        buttonAddFeed = "Feed hinzufügen",
+        buttonGoBack = "Zurückgehen",
+        buttonCancel = "Abbrechen",
+        buttonAdd = "Hinzufügen",
+        buttonChange = "Erledigt",
```

Comment:
"Erledigt" means that you have completed a specific task. "Fertig" means that you're at the end after making progress. They both can be used in the same situations, but they imply different things. For this I would go for "Fertig".